styles/widgets/material/textArea.material.less

```diff
@@ -1,5 +1,5 @@
-@TEXTAREA_CONTENT_PADDING: 10px;
-@TEXTAREA_CONTENT_LINE_HEIGHT: 24px;
+@TEXTAREA_CONTENT_PADDING: 8px;
```

Comment:
Why we change paddings here?

Reply:
My fault. I'll create specific section for this constants.